src/AnsiUtils.ps1

```diff
@@ -24,29 +24,60 @@ $AnsiEscape = [char]27 + "["
 $ColorTranslatorType = 'System.Drawing.ColorTranslator' -as [Type]
 $ColorType = 'System.Drawing.Color' -as [Type]
 
+function EscapseAnsiString([string]$AnsiString) {
```

Comment:
FYI, this function is used in the `ToString()` functions to display the original ANSI seq e.g.:
![image](https://user-images.githubusercontent.com/5177512/34470984-33056d5a-eefa-11e7-88d4-db4b3c54eec0.png)